x-pack/plugins/fleet/public/components/package_policy_actions_menu.tsx

```diff
@@ -123,7 +138,7 @@ export const PackagePolicyActionsMenu: React.FunctionComponent<{
     // </EuiContextMenuItem>,
   ];
 
-  if (!agentPolicy || !agentPolicyIsManaged) {
+  if (!agentPolicy || !agentPolicyIsManaged || agentPolicy.supports_agentless) {
```

Comment:
nit: shouldn't the last condition be `agentPolicy?.supports_agentless` as well?

Reply:
done